compose.py

```diff
@@ -315,9 +315,10 @@ def create_argmap(images):
         'Use specified Docker image to generate Docker image. Specified as <image-name>,<full-image-name>. <image-name> can be "min" or "full". Both "min" and "full" need to be specified at the same time. This will override "--container-version".'
     )
     parser.add_argument('--enable-gpu',
```

Comment:
Doesn't it still show up in --help?

Reply:
Addition of help=argparse.SUPPRESS ensures it does not show up in --help